.github/workflows/release-branch.yml

```diff
@@ -151,6 +151,8 @@ jobs:
         run: |
           echo "$INDIGO_GPG_AGENT" | base64 --decode > .key.asc
           make clean package
+      - name: Test install and uninstall
```

Comment:
This only needs to be added to the ci.yml file as that workflow will trigger the release.yml workflow